### ISSUE TITLE
Iphone SE 1st Gen.

### ISSUE THREAD

**Issue by lilboat357** [checkra1n/BugTracker#2214]
**Tell us about your setup:**
1. What iDevice are you using? (Iphone SE first gen.)
2. On what version of iOS is it? ( IOS 14.7.1)
3. What version of checkra1n are you using? (0.12.4)
4. What is your host system (OS version? Hackintosh? VM? etc.)? (hackintosh)
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? (usb A)

**What are the steps to reproduce the issue?**
1.  Load up checkra1n
2. Apply the settings for my device
2. Press next to start, and the screen freezes and doesnt do anything.

**What do you expect, and what is happening instead?**
My phone should be going into DFU/Recovery mode

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**
No.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Yes

**Any other info, error logs, screenshots, ...?** no

**Comment by zeker4321** [comment]
Saw your post on my comment. Have you tried using a different computer?

**Comment by lilboat357** [comment]
> 

Yes I have they both have the same motherboard which may be the issue. I am going to test it on my laptop.

**Comment by zeker4321** [comment]
Have you tried bootra1n on a usb stick?

**Comment by zeker4321** [comment]
Ive had luck with it in the past

**Comment by lilboat357** [comment]
Im gonna test it in a sec, well actually no my phone is disabled and I dont have a good unlocker cause it will automatically update to the newset ios
